python/lsst/ts/cbp/component.py

```diff
@@ -159,13 +172,29 @@ async def send_command(self, msg):
         """
         async with self.lock:
             msg = msg + self.terminator
-            self.log.info(f"Writing: {msg}")
+            if log:
+                self.log.info(f"Writing: {msg}")
             self.writer.write(msg.encode("ascii"))
             await self.writer.drain()
-            reply = await asyncio.wait_for(self.reader.readuntil(b"\r"), timeout=5)
-            reply = reply.decode("ascii").strip(self.terminator)
-            if reply != "":
-                self.log.info(f"reply={reply}")
+
+            if await_reply and await_terminator:
+                reply = await asyncio.wait_for(
+                    self.reader.readuntil(self.terminator.encode("ascii")), timeout=5
+                )
+            elif await_reply and not await_terminator:
+                reply = await asyncio.wait_for(self.reader.read(1024), timeout=5)
+            else:
+                reply = ":"
+
+            if reply != ":":
+                if log:
+                    self.log.info(f"reply={reply}")
```

Comment:
Does the log level here and in line 197 really have to be `info`? If the logger is a child of the CSC logger then the messages will go to EFD as well, so please consider changing to `debug`.